etc/tempesta_fw.conf

```diff
@@ -656,8 +656,8 @@
 # from HTTP tables (see 'http_chain' directive).
 #
 # <directive> is one of 'location', 'proxy_pass', 'cache_bypass', 'cache_fulfill',
-# 'nonidempotent' or 'hdr_add' directives (see the corresponding directives'
-# description).
+# 'nonidempotent', 'hdr_add' or 'http_post_validate' directives (see the
+# corresponding directives' description).
```

Comment:
Extra asterisk after `directives`

Reply:
I believe that was a "possessive 's". Since "directives" is plural, "'s" reduces to a sole apostrophe.